libs/libc/unistd/lib_getcwd.c

```diff
@@ -92,13 +98,18 @@ FAR char *getcwd(FAR char *buf, size_t size)
   /* Verify input parameters */
 
 #ifdef CONFIG_DEBUG_FEATURES
-  if (!buf || !size)
+  if (buf && !size)
```

Comment:
hmm why does is original code qualified by CONFIG_DEBUG_FEATURES?

Reply:
These parameters may be checked under DEBUG? I keep origin layout.

Reply:
If it was a error message that makes sense not a return value. The function docs state the return values and is not conditional?  Why not fix it?

Reply:
okay， done！